cli/cluster/cluster.go

```diff
@@ -341,62 +341,84 @@ func collectEtcdConfig(collectors CollectorFactory,
 	return etcdRawConfig, nil
 }
 
+type tarantoolOpts struct {
+	addr string
+	opts tarantool.Opts
+}
+
 // collectTarantoolConfig collects a configuration from tarantool config
 // storage with options from the tarantool configuration.
 func collectTarantoolConfig(collectors CollectorFactory,
 	clusterConfig ClusterConfig) (*Config, error) {
 	tarantoolConfig := clusterConfig.Config.Storage
-	var opts []connector.ConnectOpts
+	var opts []tarantoolOpts
 	for _, endpoint := range tarantoolConfig.Endpoints {
 		var network, address string
 		if !connect.IsBaseURI(endpoint.Uri) {
-			network = connector.TCPNetwork
+			network = "tcp"
 			address = endpoint.Uri
 		} else {
 			network, address = connect.ParseBaseURI(endpoint.Uri)
 		}
 
 		if endpoint.Params.Transport == "" || endpoint.Params.Transport != "ssl" {
-			opts = append(opts, connector.ConnectOpts{
-				Network:  network,
-				Address:  address,
-				Username: endpoint.Login,
-				Password: endpoint.Password,
-			})
+			opts = append(opts, tarantoolOpts{
+				opts: tarantool.Opts{
+					User:       endpoint.Login,
+					Pass:       endpoint.Password,
+					Transport:  endpoint.Params.Transport,
+					SkipSchema: true,
+				},
+				addr: fmt.Sprintf("%s://%s", network, address)})
 		} else {
-			opts = append(opts, connector.ConnectOpts{
-				Network:  network,
-				Address:  address,
-				Username: endpoint.Login,
-				Password: endpoint.Password,
-				Ssl: connector.SslOpts{
-					KeyFile:  endpoint.Params.SslKeyFile,
-					CertFile: endpoint.Params.SslCertFile,
-					CaFile:   endpoint.Params.SslCaFile,
-					Ciphers:  endpoint.Params.SslCiphers,
+			opts = append(opts, tarantoolOpts{
+				opts: tarantool.Opts{
+					User:      endpoint.Login,
+					Pass:      endpoint.Password,
+					Transport: endpoint.Params.Transport,
+					Ssl: tarantool.SslOpts{
+						KeyFile:  endpoint.Params.SslKeyFile,
+						CertFile: endpoint.Params.SslCertFile,
+						CaFile:   endpoint.Params.SslCaFile,
+						Ciphers:  endpoint.Params.SslCiphers,
```

Comment:
We need also to add SslPassword (ssl_passord) & SslPasswordFile (ssl_password_file).

Reply:
It's not yet supported by go-tarantool. I've added https://github.com/tarantool/tt/issues/744